packages/data-stores/src/woocommerce-eligibility/selectors.ts

```diff
@@ -0,0 +1,102 @@
+import { select } from '@wordpress/data';
+import { get } from 'lodash';
+import { STORE_KEY, statusMapping } from './constants';
+import { TransferEligibilityWarning, TransferEligibilityError, TransferEligibility } from './types';
+import type { State } from './reducer';
+
+export const getState = ( state: State ) => state;
+
+export const getAutomatedTransferEligibility = (
+	state: State,
+	siteId: number | null
+): TransferEligibility | null => {
+	if ( ! state || ! state.transferEligibility ) {
+		return null;
+	}
```

Comment:
`state.transferEligibility` always exists and always is an object. That's guaranteed by the store and doesn't need to be checked.